app/assets/stylesheets/application/components/work-info.scss

```diff
@@ -0,0 +1,28 @@
+.work-info {
+  @include align-items(center);
+  @include display(flex);
+
+  background-color: #f6f6f6;
+  font-size: 12px;
+  margin: 10px 0;
+  padding: 3px;
+
+  > .image {
+    @include flex-shrink(0);
+
+    display: inline-block;
+    margin-right: 3px;
+  }
+
+  > .work {
+    margin: 0 5px;
+  }
+
+  > .work > .title {
```

Comment:
Merge rule `> .work > .title` with rule on line 17<br>Selector should have depth of applicability no greater than 2, but was 3